helm/values.yaml

```diff
@@ -0,0 +1,104 @@
+# Default values for squid-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+squidConfig:
+  login: ""
+  password: ""
+  hostname: ""
+  port: 3128
+
+# Under normal circumstances one replica is needed.
+replicaCount: 1
+
+image:
+  repository: boynux/squid-exporter
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  {}
+  # fsGroup: 2000
+
+securityContext:
+  {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  className: ""
+  annotations:
+    {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
```

Comment:
Is this a correct value?

Reply:
I think yes, it should be a correct "example" value but it is up to a system engineer to decide which host to use.

My personal opinion is that Ingress might not be that necessary in many situations as a Prometheus, which is quite often deployed inside a Kubernetes cluster, should discover the ServiceMonitor and use internal Service to query the metrics. I still believe that it might be a good idea to have the optional Ingress, in case one needs to query the metrics using some external metrics collector.